packages/experimental-loader/src/stylable-runtime-loader.ts

```
@@ -27,8 +27,26 @@ const stylableRuntimeLoader: LoaderDefinition = function loader(content) {
 
     const [namespace, mapping] = evalStylableExtractModule(content);
 
+    addBuildInfo(this, namespace);
+
     return createRuntimeTargetCode(namespace, mapping);
 };
 
+function addBuildInfo(ctx: LoaderContext<any>, namespace: string) {
+    try {
+        if (ctx && ctx._module) {
```

This `if` will make it silently fail. either throw an error or remove the if